core/src/message/context.rs

```diff
@@ -61,7 +61,7 @@ impl Default for ContextSettings {
 /// Context outcome.
 ///
 /// Contains all sendings and wakes that should be done after execution.
-#[derive(Default, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Decode, Encode, TypeInfo)]
```

Comment:
This `Clone` impl and one for `MessageContext` seems excess for me, because these structs contains too much data for cloning. I want to force further development to work with it by ref by removing ability to clone